guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>Here, most of the input parameters <s>is</s> <u>are</u> similar to <s>that as</s> <u>those</u> described for <code>FullyConnectedNetwork</code>, other than the <code>preprocessors</code>. The <code>preprocessors</code> , as explained above, contains information about data transformations. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>